cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/sync/class-upload-sync.php

```diff
@@ -288,12 +288,21 @@ public function context_update( $attachment_id ) {
 	public function explicit_update( $attachment_id ) {
 		// Explicit update.
 		$type = $this->sync->get_sync_type( $attachment_id );
-		$args = $this->media->get_breakpoint_options( $attachment_id );
+		if ( wp_attachment_is_image( $attachment_id ) ) {
+			$args        = $this->media->get_breakpoint_options( $attachment_id );
+			$update_type = 'update_breakpoints';
+		} elseif ( wp_attachment_is( 'video', $attachment_id ) ) {
+			$args        = $this->media->get_pending_eagers( $attachment_id );
+			$update_type = 'update_eagers';
+		} else {
```

Comment:
No need for this else clause. Declaring `$args` and `$update_type` above the `if` is the way to go here 😄. Otherwise could get an undefined variable error for `$update_type` and the else is redundant

Reply:
🤦 - Good catch @dugajean